src/screens/PreOnboarding/PreOnboarding.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from "react";
```

Comment:
Please remove this file from this branch, you can add it in the other branch while building pre-onboarding

Reply:
yesss